modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -562,6 +562,44 @@ def test_loc_multi_index():
     df_equals(modin_df.loc[modin_df.index[:7]], pandas_df.loc[pandas_df.index[:7]])
 
 
+def test_loc_multi_index_with_tuples():
+    arrays = [
+        ["bar", "bar", "baz", "baz"],
+        ["one", "two", "one", "two"],
+    ]
+    nrows = 5
+    columns = pd.MultiIndex.from_tuples(zip(*arrays), names=["a", "b"])
+    data = np.arange(0, nrows * len(columns)).reshape(nrows, len(columns))
+    modin_df, pandas_df = create_test_dfs(data, columns=columns)
+    eval_general(modin_df, pandas_df, lambda df: df.loc[:, ("bar", "two")])
+
+
+def test_loc_multi_index_duplicate_keys():
+    modin_df, pandas_df = create_test_dfs([1, 2], index=[["a", "a"], ["b", "b"]])
+    eval_general(modin_df, pandas_df, lambda df: df.loc[("a", "b"), 0])
+    eval_general(modin_df, pandas_df, lambda df: df.loc[("a", "b"), :])
+
+
+def test_loc_multi_index_both_axes():
+    multi_index = pd.MultiIndex.from_tuples(
+        [("r0", "rA"), ("r1", "rB")], names=["Courses", "Fee"]
+    )
+    cols = pd.MultiIndex.from_tuples(
+        [
+            ("Gasoline", "Toyota"),
+            ("Gasoline", "Ford"),
+            ("Electric", "Tesla"),
+            ("Electric", "Nio"),
+        ]
+    )
+    data = [[100, 300, 900, 400], [200, 500, 300, 600]]
+    modin_df, pandas_df = create_test_dfs(data, columns=cols, index=multi_index)
+    eval_general(modin_df, pandas_df, lambda df: df.loc[("r0", "rA"), :])
+    eval_general(modin_df, pandas_df, lambda df: df.loc[:, ("Gasoline", "Toyota")])
+    # Test below bc of issue #4684
+    # df_equals(df.loc[("r0"), ("Gasoline", "Toyota")], pdf.loc[("r0"), ("Gasoline", "Toyota")])
```

Comment:
```suggestion
    # FIXME: Test below fails because of issue #4684
    # eval_general(modin_df, pandas_df, lambda df: df.loc[("r0"), ("Gasoline", "Toyota")]])
```
or, even better, parametrize the test by `.loc[]` parameter and xfail the test on one of those, like
```python
@pytest.mark.parametrize(
    "loc_label",
    [
        (("r0", "rA"), slice(None)),
        (slice(None), ("Gasoline", "Toyota")),
        pytest.param(
            (("r0"), ("Gasoline", "Toyota")),
            marks=pytest.mark.xfail("See Modin issue #4684"),
        ),
    ]
)
def test_loc_multi_index_both_axes(loc_label):
    # setup here the dataframes
    eval_general(modin_df, pandas_df, lambda df: df.loc[loc_label])
```
that way, when that issue is fixed if someone forgets to un-skip the label, CI should turn red on test which we expect to fail but it passed.

Reply:
Alternatively, I could just get rid of this test for now because I already mention it in the Github Issue.